tests/Functional/Execution/ExecutionTest.php

```diff
@@ -426,7 +426,7 @@ public function testMergeParallelFragments()
         /** @var ExecutionResult $executionResult */
         $executionResult = graphql($schema, $source);
 
-        $this->assertEquals([
+        $this->assertSame([
```

Comment:
@spawnia You have to modifier the array here in order to make two arrays identical

```
        $this->assertSame([
            'data' => [
                'a'    => 'Apple',
                'b'    => 'Banana',
                'deep' => [
                    'b'      => 'Banana',
                    'deeper' => [
                        'b' => 'Banana',
                        'c' => 'Cherry'
                    ],
                    'c'      => 'Cherry',
                ],
                'c'    => 'Cherry'
            ]
        ], $executionResult);
```

Reply:
Done